app/services/doc_auth/lexis_nexis/doc_pii_reader.rb

```diff
@@ -72,6 +82,36 @@ def parse_date(year:, month:, day:)
         Rails.logger.info(message)
         nil
       end
+
+      def parse_sex_value(sex_attribute)
+        # A value of "non-binary" or "not-specified" may appear on a document. However, at this time
+        # the DLDV `PersonSexCode` input can only process values that correspond to "male" or
+        # "female".
+        #
+        # From the DLDV User Guide Version 2.1 - 28:
+        #
+        #     Since 2017, a growing number of states have allowed a person to select "not specified"
+        #     or "non-binary" for their sex on the application for a credential. While Male and
+        #     Female can be verified, the non-binary value cannot be verified at this time.
+        #
+        # This code will return `nil` for those cases with the intent that they will not be verified
+        # against the DLDV where they will not be recognized
+        #
+        case sex_attribute&.upcase
+        when 'M'
+          'male'
+        when 'F'
+          'femaile'
+        end
+      end
+
+      def parse_height_value(height_attribute)
+        height_match_data = height_attribute&.match(/(?<feet>\d)'(?<inches>\d{1,2})"/)
```

Comment:
I am concerned about the potential in the future for a 10 foot tall person to exist

Reply:
I can adjust the regex to account for that 🤷

I am not too concerned as the [tallest person in recorded history](https://en.wikipedia.org/wiki/List_of_tallest_people) was 8'11".